### PR TITLE
Break ItemView Display & Comment into Own Packages

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
The ItemView subcomponents Display and Comment need to be consumed in terra-clinical-item-collection. ItemView.Display and ItemView.Comment have been broken out into their own component packages, such that ItemDisplay and ItemComment are now dependencies of ItemView instead of subcomponents. This allows for direct consumption.

Display is now ItemDisplay and Comment is now ItemComment. The expectations and structure of the updated ItemView component and new ItemDisplay and ItemComment components are the same as in the original ItemView implementation. 

@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-item-comment/docs/themeable-variables.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Themeable Variables
+
+Variables to `theme` the Terra Clinical ItemComment component.
+
+| Variable Name | Default Value | Usage |
+|---|---|---|
+| `$terra-clinical-item-comment-color` | `$terra-grey-70` | Sets the font color of comment.|
+| `$terra-clinical-item-comment-icon-color` | `$terra-grey-30` | Sets the color of the icon. |
```
Comment (jmsv6d) [inline packages/terra-clinical-item-comment/docs/themeable-variables.md]: Do we want to carry these over? I think we were going to hold off on adding themeable variables until we had a better idea of what actually needs to be themed.

File: packages/terra-clinical-item-display/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Terra Client Item Display
```
Comment (jmsv6d) [inline packages/terra-clinical-item-display/README.md]: *Clinical

File: packages/terra-clinical-item-view/docs/README.md [hunk 3]
```diff
@@ -19,21 +19,9 @@ import ItemView from 'terra-clinical-item-view';
   textEmphasis="default"
```
Comment (jmsv6d) [inline packages/terra-clinical-item-view/docs/README.md]: Add the imports for the ItemDisplay and ItemComment above ^

File: packages/terra-clinical-item-comment/docs/themeable-variables.md (hunk 1 above)
Comment (emilyrohrbough) [inline packages/terra-clinical-item-comment/docs/themeable-variables.md]: Good point. I removed them for now.

**Comment by emilyrohrbough** [comment]
This [commit](https://github.com/cerner/terra-clinical/pull/19/commits/b0246e2fbcdbf4350f95681de50a235bbf55233c) handles issue #18.

File: packages/terra-clinical-item-display/README.md (hunk 2 above)
Comment (emilyrohrbough) [inline packages/terra-clinical-item-display/README.md]: [Fixed Here](https://github.com/cerner/terra-clinical/pull/19/commits/1da820f20035500a114b86de22458c9211f33999#diff-741ecd409cdc6128e8ccb40d1971b00aR1)

File: packages/terra-clinical-item-view/docs/README.md (hunk 3 above)
Comment (emilyrohrbough) [inline packages/terra-clinical-item-view/docs/README.md]: [Added above](https://github.com/cerner/terra-clinical/pull/19/commits/1da820f20035500a114b86de22458c9211f33999#diff-53ff85dd3d5e28a002e5b0b44db85bd4R16)

File: packages/terra-clinical-item-display/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  - `npm install terra-clinical-item-display`
+  - `yarn add terra-clinical-item-display`
+
+## React Usage
+
+The React component can be initialized like so:
+```
+import ItemDisplay from 'terra-clinical-item-display';
+
+<Display
```
Comment (dkasper-was-taken) [inline packages/terra-clinical-item-display/docs/README.md]: ItemDisplay

**Comment by dkasper-was-taken** [comment]
It appears the styles for 'terraClinical-ItemDisplay-inlineIcon' are not being applied.

**Comment by dkasper-was-taken** [comment]
Running the site, there is a react error when attempting to display the ItemView.

**Comment by dkasper-was-taken** [comment]
You also might switch the site examples to use icons, instead of the stand-in img.

**Comment by emilyrohrbough** [comment]
@dkasper-was-taken I made the changes needed to fix everything you pointed out.

**Pull request merged**